docs/pages/setup/reference/predicate-language.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Predicate Language
+description: How to use predicate language used to define filter conditions.
+h1: Predicate Language
+---
+
+Predicate language is used to define filter or where condition for select resources.
```

Comment:
```suggestion
Teleport's predicate language is used to define conditions for filtering or selecting resources.
```

Reply:
i think my original message was misunderstood, so I modified it to this:

```
Teleport's predicate language is used to define conditions for filtering in resources.
```
unless i'm misunderstanding the use case